<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>file_selector</class>
 <widget class="QWidget" name="file_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="uiDetails">
     <property name="text">
      <string>Details</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="uiPath">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Path...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="uiBrowseBtn">
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
